### PR TITLE
Indian cookbook

### PR DESCRIPTION
**Pull request by ashGlaw**
#### Summary
```SUMMARY: Content "Adds new Indian Cookbook with recipes in"```

#### Purpose of change
This started with adding a recipe for ghee here: ```#37913```
Instead of setting its recipe to the default cookbook I wanted to add an Indian cookbook, ripe for further expansion.

#### Describe the solution
I found a book I liked the look of online and made the cookbook using it as a base. I made it somewhat rare in the itemgroups, but spawn in many. I made the curry recipes book_learn and put their recipes in here.

#### Describe alternatives you've considered
I think the idea is sound, I just feel bad for having so little content to put into it. Plenty of room for expansion

#### Testing
I spawned and read the book in game. I crafted ghee. Both worked fine

#### Additional context
This was moved to a separate PR because of the content freeze. It will have to wait.

### CODE REVIEW

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -2130,6 +2130,7 @@
     "charges": 2,
     "batch_time_factors": [ 50, 4 ],
     "autolearn": true,
+    "book_learn": [ [ "cookbook_indian", 1 ] ],
```
Comment (I-am-Erk) [inline data/json/recipes/recipe_food.json]:
```suggestion
    "book_learn": [ [ "cookbook_indian", 1 ],  [ "cookbook", 1 ] ],
```

**Comment by ashGlaw** [comment]
Cool. Good suggestion to have the recipe in both books, Erk.

Anything else needs doing?

**Comment by Shibimon** [comment]
While adding an indian cookbook its a really good idea vor variety and could lead to regional rare cookbooks, i think "ghee" that is just clarified butter is used enough in other cuisines to still be on more common books.
Butter is clarified melting it and removing the bit of liquid protein and other leftovers to keep only the pure fat and its useful in other not-indian regional foods like strudel or baklava in real life and widely used in other common foods, cause butter with liquid proteins tend to inflate and deform delicate preparations.

**Comment by ashGlaw** [comment]
Ghee and clarified butter are nearly the same thing, I agree. I considered replacing ghee with clarified butter. But this way encourages the addition of more recipes.

**Pull request merged**